doc/man7/pbkdf2.pod

```diff
@@ -0,0 +1,129 @@
+=pod
+
+=head1 NAME
+
+pbkdf2 - PBKDF2 key derivation function
```

Comment:
Agree.  Also, technically, one would want the documentation for the individual functions to be easily accessible, i.e. `man EVP_PKEY_CTX_set1_pbe_pass` should give us this page.  For that to happen, that function name must be mentioned in the NAME section.